Source/DataSources/PolygonGeometryUpdater.js

```diff
@@ -229,8 +229,10 @@ define([
     PolygonGeometryUpdater.prototype._getIsClosed = function(options) {
         var height = options.height;
         var extrudedHeight = options.extrudedHeight;
-        var isExtruded = defined(extrudedHeight);
-        return (!isExtruded && height === 0) || (isExtruded && extrudedHeight !== height && options.closeTop === true && options.closeBottom === true);
+        if (defined(extrudedHeight)) {
+            return extrudedHeight === height ? height === 0 : options.closeTop && options.closeBottom;
+        }
+        return height === 0;
```

Comment:
Again this could be simplified to: 
`return height === 0 || (defined(extrudedHeight) && extrudedHeight !== height && options.closeTop && options.closeBottom)`